plugins/supply-depot

```diff
@@ -0,0 +1,2 @@
+repository=https://github.com/Naughty-Klaus/supply-depot.git
+commit=80a76e00394a8bef64b53a6de514ba028f33eec9
```

Comment:
The plugin needs a `warning` as it makes outbound connections with the default configuration.
```suggestion
commit=80a76e00394a8bef64b53a6de514ba028f33eec9
warning=This plugin submits your username and IP address and in game stats to a 3rd party server not controlled or verified by the RuneLite Developers.
```